rclpy/src/rclpy/_rclpy.c

```diff
@@ -1356,19 +1356,31 @@ rclpy_destroy_node_entity(PyObject * Py_UNUSED(self), PyObject * args)
     Py_RETURN_FALSE;
   }
 
-  rcl_node_t * node = (rcl_node_t *)PyCapsule_GetPointer(pynode, NULL);
+  void * p = PyCapsule_GetPointer(pynode, NULL);
+  if (Py_None == p) {
```

Comment:
@dirk-thomas Just wondering why this is `Py_None == p` as opposed to `p == Py_None`. Is this style, or is this something deeper?

Reply:
This style aims to put the constant on the left to avoid accidental assignments with `=` instead of `==`.

Reply:
That makes a lot of sense! Thank you!